update-packages/update_astropy_helpers.py

```diff
@@ -1,40 +1,22 @@
-# IMPORTANT: The purpose of this script is to update the astropy-helpers
-# submodule in all affiliated packages (and a few other packages) that make use
-# of the astropy-helpers. This script is not intended to be run by affiliated
-# package maintainers, and should only be run with approval of both the
-# astropy-helpers and package-template lead maintainers, once an
-# astropy-helpers release has been made.
-
+"""
+IMPORTANT NOTICE
+================
+The purpose of this script is to update the astropy-helpers
+submodule in all affiliated packages (and a few other packages) that make use
+of the astropy-helpers. This script is not intended to be run by affiliated
+package maintainers, and should only be run with approval of both the
+astropy-helpers and package-template lead maintainers, once an
+astropy-helpers release has been made.
+
+"""
 import os
 import re
-import sys
-import shutil
-import tempfile
 import subprocess
 from distutils.version import LooseVersion
 
-from github import Github
-from common import get_credentials
-
-try:
-    HELPERS_TAG = sys.argv[1]
-except IndexError:
-    raise IndexError("Please specify the helpers version as argument")
-
-
-if LooseVersion(HELPERS_TAG) < LooseVersion('v3.0'):
-    from helpers_2 import repositories
-else:
-    from helpers_3 import repositories
-
-
-BRANCH = 'update-helpers-{0}'.format(HELPERS_TAG)
+from batchpr import Updater
 
-GITHUB_API_HOST = 'api.github.com'
-
-username, password = get_credentials()
-gh = Github(username, password)
-user = gh.get_user()
+GITHUB_TOKEN = 'fixme'
```

Comment:
I'm 👎  on this, and in fact will add the previous `get_credentials()` approach to batchpr. The changes are done locally (and used them to open those PRs for `_astropy_init.py`), but it'll need a little bit of cleanup and pushing up to batchpr.

Reply:
Indeed that is just a placeholder until `get_credentials` get added to `batchpr`. I suspect any authentication code won't be here anymore, but rather in `batchpr` in the final version of this PR.